### ISSUE TITLE
externalValue doesn't permit for relative references

### ISSUE THREAD

**Issue by spiderPan** [stoplightio/spectral#2819]
> **Please delete this section, and any sections below that you don't use, before creating the issue.**

**Describe the bug**

Starting from OAS 3.1.0, the `externalValue` should allow relative reference, See the [documentation](https://spec.openapis.org/oas/v3.1.1.html#fixed-fields-15)

**To Reproduce**

1. Create a schema as following using openapi:3.1.0
```yaml
Response:
      description: OK
      content:
        application/json:
          schema:
            $ref: '#/components/schemas/ResponseSchema'
          examples:
            jsonObject:
              externalValue: "./components/response.json"
```
2. Attempt to use spectral to validate this schema with spectral:oas rule.
3. Validation failed with 
```bash
error  oas3-schema            "externalValue" property must match format "uri".
```

**Expected behavior**
Spectral validation should pass and allow for relative URLs

**Comment by RoSk0** [comment]
Faced the same issue.

Tried three variants:
1. `relative/path.json`
2. `./relative/path.json`
3. `"./relative/path.json"`

none validates...
